tools/extra/fpgadiag/mactest.py

```diff
@@ -35,7 +35,7 @@
 
 sys_if = '/sys/class/net'
 divide = '-' * 80
-FVL_SIDE = 1
+fvl_side = 1
```

Comment:
Internal code names and their acronyms should be scrubbed from the code.

Reply:
fixed